apps/prairielearn/python/prairielearn.py

```diff
@@ -1776,8 +1776,7 @@ def load_host_script(script_name):
     """
 
     # Chop off the file extension because it's unnecessary here
-    if script_name.endswith(".py"):
-        script_name = script_name[:-3]
+    script_name = script_name.removesuffix(".py")
```

Comment:
I'm pretty impressed this change was automated 👀

Reply:
Courtesy of https://docs.astral.sh/ruff/rules/slice-to-remove-prefix-or-suffix/!